Volume/Views/BrowserView.swift

```diff
@@ -111,8 +111,10 @@ struct BrowserView: View {
 
     func displayShareScreen() {
         if let articleUrl = article.articleUrl {
-            // TODO: Allow users to share "DOWNLOAD VOLUME LINK" when we push to AppStore
-            let shareVC = UIActivityViewController(activityItems: [articleUrl], applicationActivities: nil)
+            // TODO: Add URL to secrets.plist
+            let downloadURL = URL(string: Secrets.downloadLink)
+            let shareItems: [Any] = [articleUrl, downloadURL ?? ""]
```

Comment:
Do you need to say `: [Any]` or can it be inferred?

Reply:
It actually needs the [Any] or else I get a compilation error